http.go

```diff
@@ -64,18 +63,13 @@ func (sh *ServerHandler) ServeHTTP(response http.ResponseWriter, request *http.R
 		sh.recordQueueLatencyToHistogram(s, eventType)
 	}(sh.now())
 
-	logger := sallust.Get(request.Context())
-	if logger == adapter.DefaultLogger().Logger {
-		logger = sh.log
-	}
-
-	logger.Info("Receiving incoming request...")
+	sh.log.Info("Receiving incoming request...")
```

Comment:
I think we should use `logger := sallust.Get(request.Context())` instead `sh.log` and we can remove `log                *zap.Logger` from `ServerHandler` since only `ServeHTTP` uses the logger and it's already receiving a request context